### DISCUSSION TITLE
Problem with School ID verification

### DISCUSSION THREAD

**Discussion by RaksoFox** [community/community#131945] [GitHub Education]
### Hi everyone,

I have a problem with my Polish Student ID "Legitymacja Szkolna". This document is double sided and on the first side it have School name (on stamp) and my personal information (my first and last name). On second side it have stamps for every "Valid Until" date 30 September 2023/2024 etc. I tried using OBS Virtual camera to show both sides of document at the same time because one of the isn't enough to get verified. At beginning I had multiple errors but after some corrections i ended up with 2 errors
1) Please use your device camera to submit your academic affiliation documents [...]
2) Please ensure that your academic affiliation document contains your first name exactly as it appears in your GitHub billing information [...]

I checked few times and I have same information in Billing Tab as on Student ID
After all of that i tried uploading my Student ID as "Other type of proof" but it didn't work either.

I don't have any more ideas how to get verified. Worth mentioning in previous years (2022/2023), I had no such problem because GitHub allowed uploading the file and did not force taking a photo with the device camera.

**Comment by paiineli** [comment]
Hi,

Here are a few suggestions that might help:

Check for Clear Visibility: Ensure that both sides of your ID are clearly visible and legible in the photo, as blurry or obscured images may cause errors.

Confirm Name Consistency: Double-check that your first and last name on the Student ID exactly matches the name in your GitHub billing information, including any accents or special characters.

Try Different Submission Methods: Since the direct upload option worked previously, consider reaching out to GitHub Support to explain your situation. They may be able to provide a workaround or allow you to upload your document directly.

Contact Support for Clarification: If possible, include a screenshot of the errors you're receiving when submitting, as this can help the support team understand your issue better.

I hope this helps! If you need assistance drafting a message to GitHub Support, feel free to ask.

Best,

**Comment by RaksoFox** [comment]
@paiineli How can I reach GitHub Support? I wanted to write to them now but in emails they provide only this forum for asking questions.

**Reply by yashksaini-coder** [reply]
Hi ✋🏽 @RaksoFox ,  If you are encountering any issues on GitHub and need to contact GitHub Support, you can follow these steps:-

1. **Visit the GitHub Support Page**:
   Go to the [GitHub Support page](https://docs.github.com/en/support/contacting-github-support).

2. **Choose the Appropriate Support Option**:
   GitHub offers different levels of support based on your account type (Free, Pro, Team, Enterprise, etc.). Make sure to choose the option that corresponds to your account type.

3. **Sign In to Your GitHub Account**:
   Before you can submit a support request, you need to be signed in to your GitHub account. If you don’t have an account, you’ll need to create one.

4. **Fill Out the Support Form**:
   - **Select a Category**: Choose the category that best fits your issue. Categories include Account/Billing, Security, and Technical support.
   - **Provide Details**: Clearly describe your issue. Include relevant details like error messages, steps to reproduce the issue, and any other information that might help the support team understand your problem.
   - **Attach Files**: If necessary, attach files or screenshots that can help illustrate your issue.

5. **Submit the Request**:
   Once you’ve filled out all the required fields, click the "Submit" button to send your request to GitHub Support.

For any inquiries or follow-up on the status of your request, you can check your email for updates from the GitHub Support team.

For reference, you can check out my GitHub profile [here](https://github.com/yashksaini-coder).

By following these steps, you can ensure that your issue is communicated effectively to GitHub Support, helping them to assist you more efficiently.

**Reply by RaksoFox** [reply]
Unfortunately GitHub stopped supporting manual problem solving. stopped supporting manual problem solving

![image](https://github.com/user-attachments/assets/fbf519c8-0afb-4da1-95bf-28ccab0b3b84)

**Comment by k66x1** [comment]
hey did you find a solution? we should be able to sent two photos. i were thinking about the „świadectwo szkolne” but it only shows, that i studied in previous years.
